runner/src/server/plugins/engine/views/components/clientsidefileuploadfield.html

```diff
@@ -0,0 +1,390 @@
+{% from "label/macro.njk" import govukLabel %}
+{% from "hint/macro.njk" import govukHint %}
+
+{% macro formatBytes(bytes) %}
+  {% if bytes < 1048576 %}
+    <span><strong>{{ (bytes / 1024)|round(2) }}</strong> KB</span>
+  {% elif bytes < 1073741824 %}
+    <span><strong>{{ (bytes / 1048576)|round(2) }}</strong> MB</span>
+  {% else %}
+    <span><strong>{{ (bytes / 1073741824)|round(2) }}</strong> GB</span>
+  {% endif %}
+{% endmacro %}
+
+
+{% macro ClientSideFileUploadField(component) %}
+
+<head>
+    <script src="/assets/dropzone-min.js"></script>
+    <script type="text/javascript"></script>
+</head>
+
+<style>
```

Comment:
I scoped the style specifically to this file as it's very created just for this component, and not generally re-usable.  This sort of follows the more recent design philosophy of JavaScript frameworks that have the styles, logic and templating all in one place. 

I'm open to moving it to the specific css files though, just thought it fit better here as it's very specifically for this component.